gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -203,20 +229,37 @@ def import_pangolin_vcf(vcf_path: str) -> hl.Table:
             force_bgz=True,
             reference_genome="GRCh38",
             skip_invalid_loci=True,
+            header_file=header_file_path,
         ).rows()
         return ht
 
     ht_g = import_pangolin_vcf(v4_genomes)
+    ht_g = ht_g.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes.ht", overwrite=args.overwrite
+    )
+    ht_g_new = import_pangolin_vcf(v4_genomes_new)
+    ht_g_new = ht_g_new.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes_new.ht", overwrite=args.overwrite
+    )
     ht_e = import_pangolin_vcf(v4_exomes)
+    ht_e = ht_e.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_exomes.ht", overwrite=args.overwrite
+    )
+    ht_bugfix = import_pangolin_vcf(v4_bugfix)
+    ht_bugfix = ht_bugfix.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes_bugfix.ht", overwrite=args.overwrite
+    )
```

Comment:
Since the checkpoint is run right after the import every time and goes to a 4day bucket,, lets move the checkpoint into the import function and use hl.utils.new_temp_file as the checkpoint location. 
`ht = ht.checkpoint(hl.utils.new_temp_file("pangolin", "ht"))`